ui/src/app/app.component.ts

```diff
@@ -27,10 +27,13 @@ export class AppComponent {
     }
 }
 
+/**
+ * An HTML SELECT element with a list of choices.
+ */
 @Component({
     selector: 'app-select',
```

Comment:
"app" is part of the boilerplate. I think widget-select would be a name that binds to use of widget in the layout. While widget is an outdated term, it is a ui term, and I think it is a refinement in the context of angular.